src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,163 @@
 package core.basesyntax;
 
+@SuppressWarnings("unchecked")
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] table;
+    private int size;
+    private int threshold;
+
+    static class Node<K,V> {
+        private final int hash;
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(int hash, K key, V value, Node<K,V> next) {
+            this.hash = hash;
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+
+        @Override
+        public final String toString() {
+            return key + "=" + value;
+        }
+
+        @Override
+        public final int hashCode() {
+            int result = 17;
+            result = 31 * result + (key == null ? 0 : key.hashCode());
+            result = 31 * result + (value == null ? 0 : value.hashCode());
+            return result;
+        }
+
+        @Override
+        public final boolean equals(Object o) {
```

Comment:
don't need to override these methods, it doesn't make sense because these methods are overridden in the respective K classes